### DISCUSSION TITLE
Podcast not marked as finished in iOS

### DISCUSSION THREAD

**Discussion by ict** [advplyr/audiobookshelf-app#915] [Q&A]
Does anyone else experience regular problems with Podcasts not being marked as finished and still showing up in "Resume" view after they have been completely played in the iOS app? This happens mostly when the phone is locked.

Skipping past the last minute or so (that is usually uninteresting) seems to cause it reliably.

**Comment by advplyr** [comment]
This was fixed in latest server and app https://github.com/advplyr/audiobookshelf-app/issues/893

**Reply by ict** [reply]
Thanks for the quick reply!
I'm not sure it is the same issue. I am on 0.9.66 and 2.4.4 but it is still happening. It will usually say things like "04 seconds left" for the finished episodes.

Maybe a configurable leeway on when to mark episodes as complete would make sense.
